### PR TITLE
Fix multiple apps with the same name cannot be deleted

### PR DESCRIPTION
**Pull request by bgavrilMS**
If you run the Create script multiple times, it will create apps with the same name. When you run Cleanup, the cleanup fails - this fixes the cleanup problem. 

We should also fix the Create script to check before creating new apps.

### CODE REVIEW

File: 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1 [hunk 1]
```diff
@@ -5,7 +5,7 @@ param(
     [string] $tenantId
 )
 
-if ((Get-Module -ListAvailable -Name "AzureAD") -eq $null) { 
+if ($null -eq (Get-Module -ListAvailable -Name "AzureAD")) {
```
Comment (bgavrilMS) [inline 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1]: Minor change requested by PowerShell linter

File: 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1 [hunk 2]
```diff
@@ -44,27 +44,28 @@ This function removes the Azure AD applications for the sample. These applicatio
...
     
     # Removes the applications
     Write-Host "Cleaning-up applications from tenant '$tenantName'"
 
     Write-Host "Removing 'service' (TodoListService (active-directory-dotnet-native-aspnetcore-v2)) if needed"
-    $app=Get-AzureADApplication -Filter "DisplayName eq 'TodoListService (active-directory-dotnet-native-aspnetcore-v2)'"  
+    $apps=Get-AzureADApplication -Filter "DisplayName eq 'TodoListService (active-directory-dotnet-native-aspnetcore-v2)'"  
 
-    if ($app)
+    foreach ($app in $apps)
```
Comment (bgavrilMS) [inline 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1]: Tested that we don't need the if when using the foreach...

Comment (jmprieur) [reply 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1]: so why did you keep it? for visibility, @bgavrilMS ?

Comment (bgavrilMS) [reply 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1]: I don't understand your question, I have deleted the `if ($app)` line and repalced with the `foreach`

Comment (jmprieur) [reply 1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1]: oh, I understand. We don't need the `if` if we have the `foreach`

**Pull request merged**